### ISSUE TITLE
Allow enum lower case values

### ISSUE THREAD

**Issue by mkaulig** [dillonkearns/graphqelm#12]
Using lower case enum values results in type errors in the generated code due to generated lowercase type constructors in the elm code.

The GraphQL specification seems to allow lower case enum values, although most examples use uppercase values. 
I am using graph.cool as my backend, which uses lowercase enum values in the schema by default.

Here is the corresponding part of the introspection query result:

```
{
          "inputFields": null,
          "name": "CategoryOrderBy",
          "description": null,
          "interfaces": null,
          "enumValues": [
            {
              "name": "id_ASC",
              "description": null,
              "isDeprecated": false,
              "deprecationReason": null
            },
            {
              "name": "id_DESC",
              "description": null,
              "isDeprecated": false,
              "deprecationReason": null
            },
            {
              "name": "name_ASC",
              "description": null,
              "isDeprecated": false,
              "deprecationReason": null
            },
            {
              "name": "name_DESC",
              "description": null,
              "isDeprecated": false,
              "deprecationReason": null
            }
          ],
          "fields": null,
          "kind": "ENUM",
          "possibleTypes": null
        }
```

**Comment by dillonkearns** [comment]
Fixed with with elm package version `4.0.0` and CLI version `0.0.10`. See https://github.com/dillonkearns/graphqelm/issues/13#issuecomment-355628017.

Please let me know whether this resolves the issue for you. Thanks!

**Comment by mkaulig** [comment]
This resolves the issue for me. Thanks for the fast response! 👍
